inspire/base/templates/format/record/Inspire_Default_HTML_general_macros.tpl

```diff
@@ -29,9 +29,11 @@
   {% endfor %}
   {% if record.authors|length > number_of_displayed_authors %}
   {{ sep() }}
-  <a href="#authors_{{ record['recid'] }}" class="text-muted" data-toggle="modal" data-target="#authors">
-      <em>{{ _(' Show all') }}</em>
-  </a>
+  <small>
+    <a href="#authors_{{ record['recid'] }}" class="text-muted" data-toggle="modal" data-target="#authors">
+        <em>{{ _(' Show all') }}</em>
```

Comment:
was the `Show all` link made more consistent in size with the authors as we mentioned this morning?